Modules/System/DotNet Aliases/src/dotnet.al

```diff
@@ -1990,6 +1990,9 @@ dotnet
         type("System.Security.Cryptography.Xml.XmlLicenseTransform"; "XmlLicenseTransform")
         {
         }
+        type("System.Security.Cryptography.Xml.DataObject"; DataObject)
+        {
+        }
```

Comment:
Yes. The change is, if I remember correctly, items in this file are ordered alphabetically, so I moved this in front of the KeyInfo.

But I'm not completely sure, I'm not working today, will take a look tonight

Reply:
sure, thanks!

Reply:
@maciejmlocek can you move this in front of the `KeyInfo`? Also fix the other change?

Reply:
fixed